lnwallet/channel.go

```diff
@@ -1523,16 +1581,22 @@ func (lc *LightningChannel) AddHTLC(htlc *lnwire.HTLCAddRequest) uint32 {
 	lc.ourLogIndex[pd.Index] = lc.ourUpdateLog.PushBack(pd)
 	lc.ourLogCounter++
 
-	return pd.Index
+	return pd.Index, nil
 }
 
 // ReceiveHTLC adds an HTLC to the state machine's remote update log. This
 // method should be called in response to receiving a new HTLC from the remote
 // party.
-func (lc *LightningChannel) ReceiveHTLC(htlc *lnwire.HTLCAddRequest) uint32 {
+func (lc *LightningChannel) ReceiveHTLC(htlc *lnwire.HTLCAddRequest) (uint32, error) {
 	lc.Lock()
 	defer lc.Unlock()
 
+	err := lc.Validate(lc.theirLogCounter,
+	lc.ourLogCounter, true)
```

Comment:
This line wrapping seems unnecessary. I can't tell atm as I'm not viewing this in Vim, but does this go over 80 characters? If so, then either `go-fmt` hasn't been run on this file, or Github's formatting is broked in my view.